src/aat/resources/features/F-102/S-539.td.json

```diff
@@ -0,0 +1,48 @@
+{
+  "_guid_": "S-539",
+  "_extends_": "Base_Expecting_404",
+  "title": "must return 404 for if no jurisdiction is found",
+  "specs": [
+    "a detailed profile in CCD having CR case access for a jurisdiction",
+    "returns the error message : No jurisdictions found for given access criteria",
+    "has UPDATE as case access parameter"
+  ],
+  "productName": "CCD Data Store",
+  "operationName": "Get jurisdictions available to the user",
+  "method": "GET",
+  "uri": "/aggregated/caseworkers/{uid}/jurisdictions",
```

Comment:
Can you reference F-102_Test_Data_Base.td.json?

Reply:
Updated the Json to inherit Test Base data file.